npk/tests/npk.rs

```diff
@@ -39,12 +40,12 @@ env:
   HELLO: north
 ";
 
-    fn create_test_npk(dest: &Path, manifest_name: Option<&str>) -> PathBuf {
+    fn create_test_npk(dest: &PathBuf, manifest_name: Option<&str>) -> PathBuf {
```

Comment:
Shouldn't this be `&Path`?  Think there's never a good reason to pass a immutable `&PathBuf`.